enos/enos.py

```diff
@@ -184,8 +184,15 @@ def install_os(env=None, **kwargs):
         'neutron_external_address':   pop_ip(env),
         'network_interface':          env['eths'][NETWORK_IFACE],
         'kolla_internal_vip_address': env['config']['vip'],
-        'neutron_external_interface': env['eths'][EXTERNAL_IFACE]
+        'neutron_external_interface': env['eths'][EXTERNAL_IFACE],
+        # Specific variables for the Ansible plugin
+        'influx_vip':                 env['config']['influx_vip'],
+        'kolla_ref':                  env['config']['kolla_ref']
     }
+    # Add `enable_configuring` if specified in the config file
+    if 'enable_monitoring' in env['config']:
```

Comment:
See my previous comment , this should be removed if the behavior with `hostvars` is confirmed.